### PR TITLE
zh: update monitor-a-dm-cluster.md for 2.0

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed, added or deleted? (Required)
Update monitor-a-dm-cluster.md for 2.0 from [dm.json](https://github.com/pingcap/dm/blob/master/dm/dm-ansible/scripts/dm.json)

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
preview: https://github.com/GMHDBJD/docs-dm/blob/monitor-a-dm-cluster-2.0/zh/monitor-a-dm-cluster.md

### CODE REVIEW

File: zh/monitor-a-dm-cluster.md [hunk 1]
```diff
@@ -79,12 +79,20 @@ overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分
 | process exist with error | binlog replication 在 DM-worker 内部遇到错误并且退出了 | 立即告警 | critical |
 | binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | 落后 binlog file 个数超过 1 个（不含 1 个）且持续 10 分钟时 | critical |
 | binlog file gap between relay and syncer | 与 relay 相比落后的 binlog file 个数 | 落后 binlog file 个数超过 1 个（不含 1 个）且持续 10 分钟时 | critical |
-| binlog event qps | 单位时间内接收到的 binlog event 数量 (不包含需要跳过的 event) | N/A | N/A |
-| skipped binlog event qps  | 单位时间内接收到的需要跳过的 binlog event 数量  | N/A | N/A |
-| cost of binlog event transform | Syncer 解析并且转换 binlog 成 SQLs 的耗时，单位：秒 | N/A | N/A |
+| binlog event QPS | 单位时间内接收到的 binlog event 数量 (不包含需要跳过的 event) | N/A | N/A |
+| skipped binlog event QPS | 单位时间内接收到的需要跳过的 binlog event 数量  | N/A | N/A |
+| read binlog event duration | binlog replication unit 从 relay log 或上游 MySQL 读取 binlog 的耗时，单位：秒 | N/A | N/A |
+| transform binlog event duration | binlog replication unit 解析并且转换 binlog 成 SQLs 的耗时，单位：秒 | N/A | N/A |
```
Comment (toutdesuite) [inline zh/monitor-a-dm-cluster.md:85]:
```suggestion
| transform binlog event duration | binlog replication unit 解析并且将 binlog 转换成 SQLs 的耗时，单位：秒 | N/A | N/A |
```

File: zh/monitor-a-dm-cluster.md [hunk 2]
```diff
@@ -79,12 +79,20 @@ overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分
 | process exist with error | binlog replication 在 DM-worker 内部遇到错误并且退出了 | 立即告警 | critical |
 | binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | 落后 binlog file 个数超过 1 个（不含 1 个）且持续 10 分钟时 | critical |
 | binlog file gap between relay and syncer | 与 relay 相比落后的 binlog file 个数 | 落后 binlog file 个数超过 1 个（不含 1 个）且持续 10 分钟时 | critical |
-| binlog event qps | 单位时间内接收到的 binlog event 数量 (不包含需要跳过的 event) | N/A | N/A |
-| skipped binlog event qps  | 单位时间内接收到的需要跳过的 binlog event 数量  | N/A | N/A |
-| cost of binlog event transform | Syncer 解析并且转换 binlog 成 SQLs 的耗时，单位：秒 | N/A | N/A |
+| binlog event QPS | 单位时间内接收到的 binlog event 数量 (不包含需要跳过的 event) | N/A | N/A |
+| skipped binlog event QPS | 单位时间内接收到的需要跳过的 binlog event 数量  | N/A | N/A |
+| read binlog event duration | binlog replication unit 从 relay log 或上游 MySQL 读取 binlog 的耗时，单位：秒 | N/A | N/A |
+| transform binlog event duration | binlog replication unit 解析并且将 binlog 转换成 SQLs 的耗时，单位：秒 | N/A | N/A |
```
Comment (yikeke) [inline zh/monitor-a-dm-cluster.md:85]:
```suggestion
| transform binlog event duration | binlog replication unit 解析 binlog 并将 binlog 转换成 SQL 语句的耗时，单位：秒 | N/A | N/A |
```

File: zh/monitor-a-dm-cluster.md [hunk 3]
```diff
@@ -79,12 +79,20 @@ overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分
...
+| dispatch binlog event duration | binlog replication unit 调度一条 binlog event 的耗时，单位：秒 | N/A | N/A |
+| transaction execution latency | binlog replication 执行事务到下游的耗时，单位：秒 | N/A | N/A |
+| binlog event size | binlog replication unit 从 relay log 或上游 MySQL 读取的单条 binlog event 的大小 | N/A | N/A |
+| DML queue remain length | 剩余 DML job 队列的长度 | N/A | N/A |
 | total sqls jobs | 单位时间内新增的 job 数量 | N/A | N/A |
 | finished sqls jobs | 单位时间内完成的 job 数量 | N/A | N/A |
-| execution latency | Syncer 执行 transaction 到下游的耗时，单位：秒 | N/A | N/A |
+| statement execution latency | binlog replication 执行语句到下游的耗时，单位：秒 | N/A | N/A |
```
Comment (yikeke) [inline zh/monitor-a-dm-cluster.md:92]:
```suggestion
| statement execution latency | binlog replication unit 执行语句到下游的耗时，单位：秒 | N/A | N/A |
```
少了个 unit？

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**